queries/python/indents.scm

```diff
@@ -81,35 +81,66 @@
 [
   (break_statement)
   (continue_statement)
+  (raise_statement)
+  (return_statement)
 ] @indent.dedent
 
-(parenthesized_expression ")" @indent.end)
-(generator_expression ")" @indent.end)
-(list_comprehension "]" @indent.end)
-(set_comprehension "}" @indent.end)
-(dictionary_comprehension "}" @indent.end)
+(return_statement 
+  [
+    ; indent.align
+    (list "]" @indent.dedent)
+    (dictionary "}" @indent.dedent)
+    (set "}" @indent.dedent)
+    (tuple ")" @indent.dedent)
+    (call
+      arguments: (argument_list ")" @indent.dedent))
+    (await
+      (call
+        arguments: (argument_list ")" @indent.dedent)))
+  ] 
+)
+
+(return_statement
+  [
+    ; indent.begin -> dedent twice 
+    (parenthesized_expression ")" @indent.dedent)
+    (generator_expression ")" @indent.dedent)
+    (list_comprehension "]" @indent.dedent)
+    (set_comprehension "}" @indent.dedent)
+    (dictionary_comprehension "}" @indent.dedent)
+    ; (tuple_pattern "(" @indent.begin ")" @indent.dedent)
+    ; (list_pattern "]" @indent.dedent)
+
+    ; functions
+    ; (call)
+    ; (await)
+  ]
+)
+
+(raise_statement
+  (call
+    arguments: (argument_list ")" @indent.dedent)))
+
+(parenthesized_expression 
+  ")" @indent.end
+  (#not-has-ancestor? @indent.end return_statement raise_statement))
```

Comment:
I'd prefer copy&paste to `has-ancestor?` for perf reasons.

Reply:
> I'd prefer copy&paste to `has-ancestor?` for perf reasons.

I don't exactly know how to make `has-ancestor?` works because there are so many possible other ancestors, as compared to 2 outliers
Can I use `not-has-parents?` instead? I have laid out which cases needs it but have yet to push it

Reply:
parents would be fine, way less expensive